python/lsst/ctrl/oods/gen3ButlerIngester.py

```diff
@@ -91,6 +91,39 @@ def createButler(self):
 
         return butler
 
+    def extract_info(self, dataId, key):
+        if key in dataId:
+            return f"{dataId[key]}"
+        return "??"
+
+    def extract_info_val(self, dataId, key, prefix):
+        if key in dataId:
+            return f"{prefix}%.2d" % dataId[key]
+        return f"{prefix}??"
+
+    def rawexposure_info(self, data):
+        """Return a sparsely initialized dictionary
+
+        Parameters
+        ----------
+        data: `RawFileData`
+            information about the raw file
+
+        Returns
+        -------
+        info: `dict`
+            Dictionary containing file name, and uninitialized elements
```

Comment:
I'm not sure what "uninitialized elements" means here.

Reply:
Fixed.